.flake8

```diff
@@ -1,6 +1,8 @@
 [flake8]
```

Comment:
This file should be dropped in favour of the generic `pyproject.toml` config, no?

Reply:
Can be in a seperate PR with tox config :)

Reply:
Sure, I will create another PR with pyproject.toml etc. I need to spend some time understanding all that. This one is just to get the core functionality in. I am still waiting on the Prometheus library rename fix after which I need another PR.